tests/front-end/qunit.js

```diff
@@ -0,0 +1,123 @@
+// import QUnit from "qunit";
+import idb from '../../src/js/idb';
+import Dexie from 'dexie';
+
+const db = new Dexie('test_db');
+idb.createTables(db);
+
+let tableNames = [];
+let tableKeys = [];
+
+function checkName() {
+  console.log('checkName', db.name);
```

Comment:
Can you remove this console.log please? 🙇‍♀️

Reply:
@sophielevens Done!